lib/index.js

```diff
@@ -0,0 +1,5 @@
+const foodFetches = require('./fetch-requests/foods-fetch-requests')
+
+$(document).ready( () => {
+  foodFetches.getAllFoods()
```

Comment:
This will load in both of our pages; do you have any thoughts around how we might avoid this? Not sure about this convention-wise, but one path would be for us to have two index.js files total, one for each page (foods.js, index.js)? In which case, I'd move this code out to another file.

Reply:
I mean...right now it's all getting wrapped up in the main bundle anyways.  We can absolutely break scripts apart, but that isn't something I'd want to dig into right now?  It's something we can look into, and even work with splitters in webpack? but I say, for now, stick with index.js